src/server_manager/web_app/ui_components/app-root.ts

```diff
@@ -409,7 +409,7 @@ export class AppRoot extends polymerElementWithLocalize {
             </if-messages>
             <span on-tap="maybeCloseDrawer"><a href="https://support.getoutline.org/s/article/Data-collection">[[localize('nav-data-collection')]]</a></span>
             <span on-tap="submitFeedbackTapped">[[localize('nav-feedback')]]</span>
-            <span on-tap="maybeCloseDrawer"><a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/">[[localize('nav-help')]]</a></span>
+            <span on-tap="maybeCloseDrawer"><a href="https://support.getoutline.org/">[[localize('nav-help')]]</a></span>
```

Comment:
We are `s3.amazonaws.com` to bypass the block of `getoutline.org` domain. Is it the expected behavior to switch to `getoutline.org`?

Reply:
Ah, i guess manager should be good! Thanks.